README.md

```diff
@@ -106,6 +106,12 @@ Many more features can be added by importing an existing JavaScript or CSS frame
 You need [Node](https://nodejs.org) installed on your machine to install and run Asciidoctor PDF.
 The best way to install Node is to use _nvm_ (Node Version Manager).
 
+**Install development dependencies on Linux**
```

Comment:
I think we should move it to a new "Hacking" / "Contributing code" section/document.

My concern is that it can be confusing since the "Install" section is for end-users (not developers/contributors)

Reply:
Ignore my previous comment!
We should instead add a note to tell the users that they can safely the error/warning about the optional *development* dependency `ghostscript4js` when installing `asciidoctor-pdf`.